templates/files/conf/get-vault-token

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+set -uxo pipefail
+
+# Vault configuration
+export VAULT_ADDR=https://{{ .VaultDomainName }}
+export CLUSTER_NAME={{ .ClusterName }}
+
+# Azure metadata
+jwt="$(curl -s 'http://169.254.169.254/metadata/identity/oauth2/token?api-version=2018-02-01&resource=https://management.azure.com' -H Metadata:true | jq -r '.access_token')"
+subscription_id=$(curl -s -H Metadata:true "http://169.254.169.254/metadata/instance?api-version=2017-08-01" | jq -r '.compute | .subscriptionId')
+vm_name=$(curl -s -H Metadata:true "http://169.254.169.254/metadata/instance?api-version=2017-08-01" | jq -r '.compute | .name')
```

Comment:
potentially VM name could have spaces in it, maybe it is better to put double quotes around this variable

```suggestion
vm_name="$(curl -s -H Metadata:true "http://169.254.169.254/metadata/instance?api-version=2017-08-01" | jq -r '.compute | .name')"
```

Reply:
vm names are master/worker. So we don't have spaces there for sure. But just to be safe - sure